packetery/src/Packetery/Order/class-wrapper.php

```diff
@@ -0,0 +1,133 @@
+<?php
+/**
+ * Class Entity
+ *
+ * @package Packetery\Order
+ */
+
+declare( strict_types=1 );
+
+
+namespace Packetery\Order;
+
+/**
+ * Class Entity
+ *
+ * @package Packetery\Order
+ */
+class Wrapper {
```

Comment:
Pokud je to entita, pojmenoval bych třídu "Order" a ne "Wrapper".

Reply:
Takže místo \Packetery\Order\Wrapper použít \Packetery\Order\Order ? A co \Packetery\Order\Entity ? V kódu se může použít "Order\Entity".